kilkari/src/main/java/org/motechproject/nms/kilkari/domain/CallSummaryRecord.java

```diff
@@ -10,7 +10,7 @@
 import javax.validation.constraints.Min;
 import java.util.Map;
 
-@Entity(tableName = "nms_kilkari_call_summary_records")
+@Entity(tableName = "nms_kk_summary_records")
```

Comment:
unrelated but looks like we have a few different naming schemes we follow for the domain entity tables...even within the same module